README.md

```diff
@@ -1,54 +1,75 @@
-# Evervault
-<p align="center">
-  <img src="res/logo.svg">
+<p>
+  <a href="https://welcome.evervault.com/"><img src="res/logo.svg"></a>
 </p>
 
-<p align="center">
+<p>
   <a href="https://github.com/evervault/evervault-ruby/actions?query=workflow%3Aevervault-unit-tests"><img alt="Evervault unit tests status" src="https://github.com/evervault/evervault-ruby/workflows/evervault-unit-tests/badge.svg"></a>
 </p>
 
+# Evervault Ruby SDK
+
+The [Evervault](https://evervault.com) Ruby SDK is a toolkit for encrypting data as it enters your server, and working with Cages.
 
 ## Getting Started
-Ruby SDK for [Evervault](https://evervault.com)
-### Prerequisites
 
-To get started with the Evervault Ruby SDK, you will need to have created a team on the Evervault dashboard.
+Before starting with the Evervault Ruby SDK, you will need to [create an account](https://app.evervault.com/register) and a team.
+
+For full installation support, [book time here](https://calendly.com/evervault/cages-onboarding).
 
-We are currently in invite-only early access. You can apply for early access [here](https://evervault.com).
+## Documentation
+
+See the Evervault [Ruby SDK documentation](https://docs.evervault.com/ruby).
 
 ## Installation
 
-Add this line to your application's Gemfile:
+There are two ways to install the Ruby SDK.
+
+#### 1. With Gemfile
+
+Add this line to your application's `Gemfile`:
 
 ```ruby
 gem 'evervault'
 ```
 
-And then execute:
+Then, run:
+
 ```sh
-     bundle install
+bundle install
 ```
-Or install it yourself as:
+#### 2. By yourself
+
+Just run:
+
 ```sh
-     gem install evervault
+gem install evervault
 ```
 
 ## Setup
 
-Evervault can be initialized as a singleton throughout the lifecycle of your application.
+There are tw ways to make Evervault available for use in your app.
+
+#### 1. As a singleton
+
+The singleton pattern is recommended because eusing your Evervault client prevents additional overhead of loading keys at Cage runtime.
```

Comment:
```suggestion
The singleton pattern is recommended to prevent additional overhead of loading keys at Cage runtime when creating new Clients.
```
I think this reads better - `because` doesn't tie the sentence together well imo.